bindings/node/index.d.ts

```diff
@@ -203,7 +203,13 @@ export interface ClientEncryptionOptions {
   /**
    * TLS options for kms providers to use.
    */
-  tlsOptions?: ClientEncryptionTLSOptions;
+  tlsOptions?: {
```

Comment:
TLS options to KMS providers are configured on a per KMS provider basis for future flexibility.